### PR TITLE
[MIRROR] paraplegic vehicle fixes

### PR DESCRIPTION
**Pull request by ghost**
Original PR: https://github.com/tgstation/tgstation/pull/43399
----------------------------
:cl: ShizCalev
fix: Paraplegics/mobs missing legs can no longer operate vehicles which require legs to operate (ie bicycles, skateboards, ATVs, ect) if they lost the limbs while still buckled in.
tweak: Mobs with no arms will now fall off bikes & scooters when trying to move.
tweak: Badmins can now edit limb requirements of vehicles.
/:cl:

### CODE REVIEW

File: code/modules/vehicles/scooter.dm [hunk 1]
```diff
@@ -2,6 +2,8 @@
 	name = "scooter"
 	desc = "A fun way to get around."
 	icon_state = "scooter"
+	are_legs_exposed = TRUE
+	fall_off_if_missing_arms = TRUE
```
Comment (Jujumatic) [inline code/modules/vehicles/scooter.dm:6]: yet another 2 vars that we don't have on our codebase

what the fuck?

Comment (mikolaj007) [reply code/modules/vehicles/scooter.dm]: There was a recent vehicle fix thingy, thats why the vars are there

**Comment by Jujumatic** [comment]
i believe we missed a mirror as the vehicles have vars that have not been defined anywhere

**Comment by ghost** [comment]
did we not merge tg's version of the wheelchair

**Review by yoyobatty — DISMISSED** [review]

**Review by yoyobatty — APPROVED** [review]

**Pull request merged**